service-front/module/Application/view/application/authenticated/dashboard/macros.twig

```diff
@@ -15,13 +15,13 @@
             <li class="list-item list-item--v3 status-container--{{ status }}" {% if lpa.refreshId %}data-refresh-id="{{ lpa.refreshId }}"{% endif %}>
                 <div class="info">
                     <div class="main-info">
-                        <h1 class="list-item_title">
+                        <h3 class="list-item_title">
```

Comment:
Quick one, @Raffers: is this list always displayed under an h2? I think it follows a h1, so should be an h2, otherwise that introduces a different accessibility issue (incorrect heading numbering).

Reply:
The list is always displayed under an `h2` element, just double checked.